corehq/apps/fixtures/upload/upload.py

```diff
@@ -182,8 +182,9 @@ def get_all_type_sheets(self):
         return type_sheets
 
     def validate(self):
+        from corehq.apps.fixtures.upload.run_upload import validate_fixture_upload
         self.get_types_sheet()
-        self.get_all_type_sheets()
```

Comment:
was this just running a check that was also run in what became `validate_fixture_upload`?